_includes/header.html

```diff
@@ -2,7 +2,13 @@
 
   <div class="wrapper">
 
-    <a class="site-title" href="{{ site.baseurl }}/">{{ site.title | escape }}</a>
+    <a class="site-title" href="{{ site.baseurl }}/">
+      {% if site.logo %}
+        <img src="/{{ site.logo }}" class="site-logo" width="150" height="auto">
```

Comment:
For this to not break the site header layout you should replace the width attribute with a `height="56"` attribute.